bug-bounty.md

```diff
@@ -0,0 +1,43 @@
+# Bug Bounty
+
+The Moonbeam bug bounty program is focused on the Moonriver and Moonbeam Parachains (deployed to Kusama and Polkadot respectively) and dAapps. It is focused on preventing:
+
+* Thefts and freezing of principal of any amount
+* Thefts and freezing of unclaimed yield of any amount
+* Theft of governance funds 
+* Governance activity disruption
+* Network shutdown
+* Website goes down
+* Leak of user data
+* Deletion of user data
+* Access to sensitive pages without authorization
```

Comment:
Some of the terminology here is a bit confusing as it could relate to both our blockchain or website/other infrastructure. Maybe splitting this out into two or more categories would help?

Reply:
Updated, thanks!